src/ESPiLight.cpp

```diff
@@ -154,6 +154,7 @@ ESPiLight::ESPiLight(int8_t outputPin) {
   _outputPin = outputPin;
   _callback = nullptr;
   _rawCallback = nullptr;
+  _echoEnabled = true;
```

Comment:
Would be fine to have it disable by default. It is basically for debugging only.